src/bioregistry/data/bioregistry.json

```diff
@@ -4496,7 +4496,6 @@
     }
   },
   "bko": {
-    "banana": "BKO:",
```

Comment:
Still confused about this... as far as I could tell, the BKO: prefix is shown explicitly everywhere an ID appears on the provider website, as well as in the URL pattern the website uses to resolve IDs in their native ontology browser.